frontend/src/global_styles/layout/_colors.sass

```
@@ -95,8 +95,8 @@
   font-weight: var(--base-text-weight-bold)
 
 // Inline: all except type
-[class^='__hl_inline_']:not([class^='__hl_inline_type']),
-[class*=' __hl_inline_']:not([class*='__hl_inline_type'])
+[class^='__hl_inline_']:not([class^='__hl_inline_type'], [class^='__hl_inline_life_cycle_step_definition']),
+[class*=' __hl_inline_']:not([class*='__hl_inline_type'], [class^='__hl_inline_life_cycle_step_definition'])
```

I believe the correct version would have been using the `class*` for the life cycle step definition.
```suggestion
[class*=' __hl_inline_']:not([class*='__hl_inline_type'], [class*='__hl_inline_life_cycle_step_definition'])
```